aip/0135.md

```diff
@@ -96,7 +96,8 @@ resource (not `google.protobuf.Empty`), with its [state][aip-216] set to
 `DELETED`.
 
 Soft-deleted resources **should not** show up in [list][aip-132] requests by
-default (unless `bool show_deleted` is true).
+default (unless `bool show_deleted` is true). Soft-deleted resources **should**
```

Comment:
"should show up in" sounds weird for a request for a single resource. Perhaps:

> `Get` requests for soft-deleted resources **should** return the resource (rather than a `NOT_FOUND` error).

Reply:
Done.